### PR TITLE
Use the FQCN throughout the whole project

### LINKED ISSUE #12: Use FQCN everywhere in the project
This way we're compatible with the most recent `ansible-lint` release as prepared in #7.

### PR DESCRIPTION
**Pull request by tobiashuste**
Closes #12 
Required for #7

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -70,27 +70,27 @@
 
     - name: Build Redis.
       become: yes
-      make:
+      ansible.builtin.make:
```
Comment (christianhueserhzdr) [inline tasks/main.yml:73]: Dear Tobias @tobiashuste, I think this should be the same as the follow-up make call:

```suggestion
      community.general.make:
```

**Review by christianhueserhzdr — CHANGES_REQUESTED** [review]
Dear Tobias @tobiashuste, I spotted one inconsistency regarding make calls in the role tasks. All else is fine.

File: tasks/main.yml (hunk 1 above)
Comment (tobiashuste) [reply tasks/main.yml:73]: Thanks, I missed that!

**Review by christianhueserhzdr — APPROVED** [review]
Dear Tobias @tobiashuste, thanks for fixing the FQCN of the respective make call. I approve and merge this PR now.

**Pull request merged**